readme.md

```diff
@@ -68,8 +68,8 @@ _Former libraries that are still supported._
 
 _Libraries that are no longer supported. We still accept patches._
 
-- [can-ejs](/canjs/can-ejs)
-- [can-mustache](/canjs/can-mustache)
+- [can-ejs](https://v2.canjs.com/docs/can.ejs.html)
+- [can-mustache](https://v2.canjs.com/docs/can.mustache.html)
```

Comment:
These should link to what's in http://canjs.com/doc/can-legacy.html

Remove `can-mustache`.